From 0fc532d3938086c441fbd6b0a20983f70341c5e9 Mon Sep 17 00:00:00 2001
From: "Reddy, Alavala Srinivasa" <alavala.srinivasa.reddy@intel.com>
Date: Mon, 14 Aug 2023 11:39:59 +0530
Subject: [PATCH] Fix for ASB-Aug-2023-build-error.

Removal of below API causing build error,
as API is present in the code base

Signed-off-by: Reddy, Alavala Srinivasa <alavala.srinivasa.reddy@intel.com>
---
 .../android/server/policy/keyguard/KeyguardStateMonitor.java  | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/services/core/java/com/android/server/policy/keyguard/KeyguardStateMonitor.java b/services/core/java/com/android/server/policy/keyguard/KeyguardStateMonitor.java
index 00ed8de38c4f..c0aa8aeff711 100644
--- a/services/core/java/com/android/server/policy/keyguard/KeyguardStateMonitor.java
+++ b/services/core/java/com/android/server/policy/keyguard/KeyguardStateMonitor.java
@@ -82,10 +82,6 @@ public class KeyguardStateMonitor extends IKeyguardStateCallback.Stub {
         return mCurrentUserId;
     }
 
-    public int getCurrentUser() {
-        return mCurrentUserId;
-    }
-
     @Override // Binder interface
     public void onShowingStateChanged(boolean showing, int userId) {
         if (userId != mCurrentUserId) return;
-- 
2.17.1

